### ISSUE TITLE
Support for Gatsby-plugin-image

### ISSUE THREAD

**Issue by frodeste** [Chuloo/gatsby-source-cloudinary#29]
Is support for gatsby-plugin-image planned?

**Comment by miguelt1** [comment]
I would love to have that feature too

**Comment by maplessmann** [comment]
I'm also interested in this feature. I'm trying to use this plugin, but it doesn't work with `gatsby-plugin-image` 😞

**Comment by ptimson** [comment]
Hey all! I've just hit this problem myself and found a solution. Use this plugin in conjunction with [`gatsby-plugin-remote-images` ](https://github.com/graysonhicks/gatsby-plugin-remote-images) 

All you need to do install both plugins and configure like so:

```
 {
  resolve:`gatsby-source-cloudinary`,
  options: {
    cloudName: process.env.CLOUDINARY_CLOUD_NAME,
    apiKey: process.env.CLOUDINARY_API_KEY,
    apiSecret: process.env.CLOUDINARY_API_SECRET,
    resourceType: `image`,
  }
},
{
  resolve: `gatsby-plugin-remote-images`,
  options: {
    nodeType: 'CloudinaryMedia',
    imagePath: 'url',
  },
},
```

You will then get a `localFile` variable added to your `allCloudinaryMedia` and `cloudinaryMedia` queries that includes the 
`gatsby-plugin-image`

**Comment by raae** [comment]
Support coming through gatsby-transformer-cloudinary@^3.0.0 (now in beta) + soon to be released gatsby-source-cloudinary@*0.2.0 🎉
